meerket/meerket-application/src/main/java/org/j1p5/api/comment/dto/response/CommentReadResponseDto.java

```diff
@@ -0,0 +1,34 @@
+package org.j1p5.api.comment.dto.response;
+
+import org.j1p5.api.comment.dto.CommentMemeberDto;
+import org.j1p5.domain.comment.entity.CommentEntity;
+import org.j1p5.domain.user.entity.UserEntity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public record CommentReadResponseDto(
+        CommentMemeberDto commentMemeberDto,
+        Long commentId,
+        String content,
+        boolean isSeller,//판매자인지
+        boolean isUpdatable,//수정된 글인지
```

Comment:
```suggestion
        boolean isUpdated
```
가 수정된 글인지 유무 판단하기에 좋은 것 같습니다!
이전 api 명세에 저렇게 적은 것 같은데 명확한 의미 전달을 위해 수정하는 게 좋을 것 같아요!